protostar/commands/test/test_command.py

```diff
@@ -122,6 +128,7 @@ async def test(
         no_progress_bar=False,
         fast_collecting=False,
         exit_first=False,
+        stdout_on_success=False,
```

Comment:
```suggestion
        stdout_on_success: bool = False,
```

Reply:
We don't do that anywhere else, is it necessary here?